docs/components/AnnouncementBubbleView.jsx

```diff
@@ -116,6 +116,35 @@ export default class AnnouncementBubbleView extends React.PureComponent {
               Hello class! Links like https://clever.com are clickable
             </AnnouncementBubble>
 
+            <AnnouncementBubble
+              className={cssClass.BUBBLE}
+              readBy={[
+                "Arsalan",
+                "Nikhil",
+                "Spencer",
+                "Ashley",
+                "Jonathan",
+                "Brian",
+                "Chloe",
+                "Cory",
+                "Jonah",
+                "Nick",
```

Comment:
Oh I didnt mean to tell you that you had to change it, i thought it was funny :D but this is also great! haha. I love easter eggs in our code